src/main/config/config.c

```diff
@@ -232,14 +217,29 @@ void validateAndFixConfig(void)
         featureClear(FEATURE_RX_SERIAL | FEATURE_RX_MSP | FEATURE_RX_PPM);
     }
 
+    // The retarded_arm setting is incompatible with pid_at_min_throttle because full roll causes the craft to roll over on the ground.
+    // The pid_at_min_throttle implementation ignores yaw on the ground, but doesn't currently ignore roll when retarded_arm is enabled.
+    if (armingConfig()->retarded_arm && mixerConfig()->pid_at_min_throttle) {
+        mixerConfig()->pid_at_min_throttle = 0;
+    }
+
+
 #ifdef STM32F10X
     // avoid overloading the CPU on F1 targets when using gyro sync and GPS.
     if (imuConfig()->gyroSync && imuConfig()->gyroSyncDenominator < 2 && featureConfigured(FEATURE_GPS)) {
         imuConfig()->gyroSyncDenominator = 2;
     }
 #endif
+#ifdef STM32F303xC
+    // hardware supports serial port inversion, make users life easier for those that want to connect SBus RX's
+#ifdef TELEMETRY
+    telemetryConfig()->telemetry_inversion = 1;
```

Comment:
@hydra , I did not add the line `telemetryConfig()->telemetry_inversion = 1;`, I just moved it - the behaviour in this respect is unchanged.

While this is a bug, it is, as @ledvinap stated, unrelated to this PR.

So this PR should be merged and a new issue raised.